qiskit/providers/ibmq/runtime/__init__.py

```diff
@@ -213,6 +214,8 @@ def interim_result_callback(job_id, interim_result):
    UserMessenger
    ProgramBackend
    ResultDecoder
+   RuntimeEncoder
+   RuntimeDecoder
```

Comment:
You should add the new `ParameterNamespace` class here as well so its methods get picked up in the [API doc](https://qiskit.org/documentation/apidoc/ibmq-provider.html).

Reply:
Fixed!